### DISCUSSION TITLE
Github copilot not listed in martket place in pycharm

### DISCUSSION THREAD

**Discussion by lualeperez** [community/community#7735] [Copilot Conversations] [answered]
Hello all,

I tried to install the plugin for github copilot in pycharm (jetbrains) following the instruction in here,
https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md#getting-started-with-github-copilot-in-jetbrains

I put "gihub copilot" in the search box for the marketplace and I get this (see attached file). As you can see, the github copilot is not listed, but Tabnine AI code completion.

Any help will be very much appreciated.

Regards,

  Alejandro

<img width="1209" alt="Screenshot 2021-11-15 at 16 06 52" src="https://user-images.githubusercontent.com/43741533/141805063-2e885411-c5d4-445e-b323-d2d2f5375415.png">

**Comment by jansorg** [comment, marked as answer]
Are you using PyCharm 2021.2 or later? Version 2021.2 is the earliest supported version.

**Reply by lualeperez** [reply]
I update to PyCharm version 2021.3 and it is still not listed in the marketplace

**Reply by jansorg** [reply]
That's odd. Do you have any custom plugin repositories configured?
Alternatively, you could download https://plugins.jetbrains.com/plugin/17718-github-copilot/versions and install manually by Drag&Drop. But, most likely, you still won't get updates unless your original problem is solved.

Here's how it look with PyCharm Community 2021.3 rc.
![image](https://user-images.githubusercontent.com/11473063/142843389-81d06817-074c-4a4c-b307-b33eb77baaa7.png)

**Reply by adiSilber** [reply]
I have the same problem, but I am not sure that I updated my Pycharm since I did enabled "Update all tools automatically" in JetBrains toolbox but under Settings -> appearance and behavior -> System Settings -> Updates I see that my current version is PyCharm 2019.2.4. can any of you help with that? thanks!

**Reply by jansorg** [reply]
@adiSilber 2019.2 is too old. Launching JetBrains Toolbox and clicking the update button next to PyCharm should be enough.

**Comment by haoyang1166** [comment]
I also hava this problem

**Reply by haoyang1166** [reply]
i use newest version

**Comment by rbrook** [comment]
Make sure you're on the "Marketplace" tab in "Plugins" and not "Installed"
